components/typography/typography-styles.js

```diff
@@ -0,0 +1,85 @@
+import { css } from 'lit-element/lit-element.js';
+
+export const typographyStyles = css`
```

Comment:
These styles are translated from the original mixins.  The expectation is that other components would import this stylesheet and include in their static styles getter.  I was considering breaking these up so that components can include what it wants - for example, body text, headings, label.  Thoughts?

I nice thing about this approach (in contrast to the mixins) is that the corresponding media queries can be included here, whereas previously the consuming component had to deal with that itself.

Reply:
I totally agree. And yeah, I think breaking these up makes sense, since in theory including more than you need means potentially more evaluations.